scripts/cc-cli/src/test/integration-tests/collectCoins.test.ts

```diff
@@ -0,0 +1,116 @@
+import { newApi } from "../../api";
+import { initKeyringPair } from "../../utils/account";
+import { parseAmountInternal } from "../../utils/parsing";
+import { signSendAndWatch } from "../../utils/tx";
+import {
+    fundAddressesFromSudo,
+    fundFromSudo,
+    randomTestAccount,
+} from "./helpers";
+import { arg } from "../../globalSetup";
+import execa from "execa";
+import {
+    ApiPromise,
+    Blockchain,
+    CreditcoinApi,
+    KeyringPair,
+    Wallet,
+    creditcoinApi,
+    providers,
+} from "creditcoin-js";
+import { cryptoWaitReady } from "@polkadot/util-crypto";
+import { AUTHORITY_SURI } from "creditcoin-js/lib/examples/setup-authority";
+import { testData, tryRegisterAddress } from "creditcoin-js/lib/testUtils";
+
+describe("collect-coins", () => {
```

Comment:
integration-tests/src/utils.ts defines a `describeIf` function which will skip the entire section if you can't execute the sudo transaction. Will save you some ifconditions down below.

Reply:
Any opposition to me moving `testIf` and `describeIf` into `creditcoin-js`?

Reply:
No, however that's probably best to be a separate PR as it will introduce other unrelated changes after itself.